REQUIRE

```diff
@@ -1,5 +1,5 @@
 julia 0.5
-DataArrays 0.3.4
+DataArrays 0.5
```

Comment:
If you revert this part do the tests pass on 0.5 and 0.6? The code you've changed here is necessary for DataArrays on 0.6 but should also work on 0.5 (the older DataArrays code just isn't optimized for that style), so we might be able to get away with letting dependency resolution do its thing in this case.